ibm_vpd_app.cpp

```diff
@@ -918,6 +923,26 @@ static void populateDbus(T& vpdMap, nlohmann::json& js, const string& filePath)
         const auto& objectPath = item["inventoryPath"];
         sdbusplus::message::object_path object(objectPath);
 
+        vector<string> ccinList;
+        if (item.find("ccin") != item.end())
+        {
+            for (const auto& cc : item["ccin"])
+            {
+                string ccin = cc;
+                transform(ccin.begin(), ccin.end(), ccin.begin(), ::toupper);
```

Comment:
I like how you are making developers lives easier here by anticipating a silly mistake in the json.  nice!